python/pyspark/sql/functions.py

```diff
@@ -5883,6 +5883,28 @@ def posexplode(col: "ColumnOrName") -> Column:
     return _invoke_function_over_columns("posexplode", col)
 
 
+def inline(col: "ColumnOrName") -> Column:
```

Comment:
Should be listed in https://github.com/apache/spark/blob/master/python/docs/source/reference/pyspark.sql/functions.rst

Reply:
Added to that file